lib/models/index.js

```diff
@@ -50,6 +50,8 @@ db.participant.belongsTo(db.cause)
 db.event.belongsToMany(db.participant, {through: 'participant_event'})
 db.participant.belongsToMany(db.event, {through: 'participant_event'})
 
+db.fcmtoken.belongsTo(db.participant)
```

Comment:
It would probably be helpful to add the reverse of this above, e.g. `db.participant.hasMany(db.fcmtoken)`